### ISSUE TITLE
Playing audio files in HTML

### ISSUE THREAD

**Issue by rsuhaibani** [FrozenSquirrels/polyglot#34]
Here a simple code for playing audio files in HTML5
```
<audio controls>
  <source src="horse.mp3" type="audio/mpeg">
Your browser does not support the audio element.
</audio>
```
@hananmuhajab @nathanbreitsch We need to try it with the audio records that we are having...

**Comment by rsuhaibani** [comment]
The code is working fine..
Here is how the result should look like in the web page:
![screen shot 2015-04-21 at 1 38 15 am](https://cloud.githubusercontent.com/assets/8633667/7246193/4d0219b6-e7c7-11e4-8adf-f8e28d5b33ee.png)

**Comment by hananmuhajab** [comment]
Do you add anew column in dataset ??

**Comment by rsuhaibani** [comment]
Not yet

**Comment by hananmuhajab** [comment]
@rsuhaibani @nathanbreitsch 
I tried to add the audio , it works and appears just in case we don't import the dataset , but when I imported the dataset , it appears error because the sound column  not as a part dataset

**Comment by rsuhaibani** [comment]
Yes as we discussed earlier, we should import the sounds URLs besides the words dataset as a suggested solution. So we need to upload at least one sound to try that... 
I did upload the sounds to our directory under a folder called sounds, but it did not show up and I do not know why...at the same time I have sent all the mp3 recordings to both of you..
As for now, I will upload one recordings on a free hosting and post the link here..

**Comment by rsuhaibani** [comment]
Here is one link: http://s000.tinyupload.com/download.php?file_id=04690372659236688990&t=0469037265923668899009324

If this solution worked, we will upload all the recordings on our domain then use the resulted URLs for our dataset..

**Comment by hananmuhajab** [comment]
unfortunately, this way will not work because as you know this kind of  importing in Mongo will causes this error   " the Invalid UTF-8 character detected" since the link has some unavailable character.

**Comment by rsuhaibani** [comment]
You are right. So, @dverhas or @nathanbreitsch please upload the recordings to our domain so we can find a way for importing them into mongodb instead of using a long link with invalid characters...

**Comment by rsuhaibani** [comment]
Useful Links for storing videos/audios in mongodb: 
1. http://www.mongodb.com/blog/post/building-mongodb-applications-binary-files-using-gridfs-part-1
2. https://www.mongodb.com/blog/post/building-mongodb-applications-binary-files-using-gridfs-part-2
